app/views/instruments/public_schedule.html.haml

```diff
@@ -23,6 +23,9 @@
 
 %h2 #{@instrument} #{t('.title')}
 
+- unless @instrument.online?
```

Comment:
Would it be worth adding an `Instrument#offline?` method that's just the inverse of `online?` so we can make these `if`s?